src/guides/v2.3/javascript-dev-guide/javascript/js_init.md

```diff
@@ -133,6 +133,9 @@ require([
 </script>
 ```
 
+{:.bs-callout-tip}
+For avoid miss initialize scripts we recommend all developers should use declarative syntax. Both Declarative and Imperative will work, Depends on real specific requirements choose wisely
```

Comment:
Please re-write these two sentences as they are not very clear English and correcting them might change their meaning.